pkg/web/templates/partials/secure_envelope/secure_envelope_detail.html

```diff
@@ -1,9 +1,84 @@
-{{ if .SecureEnvelope.Payload }}
-{{ range .SecureEnvelope.Payload }}
-<div>{{ . }}</div>
-{{ end }}
-{{ end }}
-
-{{ if .SecureEnvelope.Error }}
-<p>{{ .SecureEnvelope.Error }}</p>
-{{ end }}
\ No newline at end of file
+<input type="radio" name="my-accordion-1" checked="checked" />
+{{ if .IsError }}
+<div class="collapse-title bg-red-200">
+  {{ else }}
+  <div class="collapse-title">
+    {{ end }}
+    <div class="flex items-center gap-x-1 pb-2 lg:text-lg font-semibold">
+      {{ if eq .Direction "in" }}
+      <h3>Incoming:</h3>
```

Comment:
The accordion component uses a radio input to toggle open and close. When I added an icon to the title the tooltip didn't appear. When I removed the input I was able to see the tooltip, so it seems that the checked state is somehow blocking the tooltip from appearing in the UI. This is why I displayed the text for now since that's the quicker fix than replacing the component at the moment.